pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -1980,6 +2003,25 @@ func (service *SecurityPolicyService) buildRuleHashString(rule *v1alpha1.Securit
 	return util.Sha1(string(serializedBytes))
 }
 
+func (service *SecurityPolicyService) buildVPCRuleID(obj *v1alpha1.SecurityPolicy, ruleHash string, ruleIdExistsFn func(id string) bool) string {
+	spUID, suffixInUid := parseSuffixInUid(obj.GetUID())
```

Comment:
I can't figure out what `suffixInUid` would look like. Could you please give an example?